src/app/models/parsed-elements.ts

```diff
@@ -22,4 +22,8 @@ export type NoteData = GenericElementData & {
     exponent: string;
 };
 
+export interface ParagraphData extends GenericElementData {
+    n: string;
```

Comment:
What does `n` represents?

Reply:
`n` is the number of the paragraph. It can be defined in the xml, or it can be automatically created. For the moment we need to handle only the first case.